vlfm/vlm/server_wrapper.py

```diff
@@ -32,7 +32,8 @@ def process_request() -> Dict[str, Any]:
         payload = request.json
         return jsonify(model.process_payload(payload))
 
-    app.run(host="localhost", port=port)
+    app.run(host="0.0.0.0", port=port) # app.run(host="localhost", port=port)
```

Comment:
this is for external access?

Reply:
This opens up the flask server to the local network. Still allows access to localhost so doesn't need any changes to any other code if models are still hosted on the device connected to spot.

But also allows for the option to host models elsewhere on the network allowing for bigger and better models while having smaller edge compute devices on the spot instead of needing all the compute to be on or near spot. Opens up the option to easily add more models of all sizes if interested